pkg/agent/flowexporter/ipfix/ipfix_process.go

```diff
@@ -63,31 +61,21 @@ func (exp *ipfixExportingProcess) CloseConnToCollector() {
 }
 
 func (exp *ipfixExportingProcess) LoadRegistries() {
-	exp.ianaReg = ipfixregistry.NewIanaRegistry()
-	exp.ianaReg.LoadRegistry()
-	exp.antreaReg = ipfixregistry.NewAntreaRegistry()
-	exp.antreaReg.LoadRegistry()
+	ipfixregistry.LoadRegistry()
 	return
 }
 
-func (exp *ipfixExportingProcess) GetIANARegistryInfoElement(name string, isReverse bool) (*ipfixentities.InfoElement, error) {
+func (exp *ipfixExportingProcess) GetInfoElement(name string, isReverse bool, enterpriseID uint32) (*ipfixentities.InfoElement, error) {
```

Comment:
isReverse is not required anymore I guess?

Reply:
Makes sense. Thanks!